spec/views/group_forms/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "rails_helper"
+
+RSpec.describe "group_forms/edit.html.erb", type: :view do
+  let(:group) { create(:group) }
+  let(:form) { create :form, id: 1 }
+  let(:group_select) { Forms::GroupSelect.new(group: group, form: form) }
+  let(:group_select_presenter) { Forms::GroupSelectPresenter.new(form: form, group: group, groups: group_select.groups) }
+
+  before do
+    group.group_forms.build(form_id: form.id)
+    group.save!
+
+    assign(:id, form.id)
+    assign(:form, form)
+    assign(:group, group)
+    assign(:group_form, group.group_forms.first)
+    assign(:group_select, group_select)
+    assign(:group_select_presenter, group_select_presenter)
+  end
+
+  it "renders the page title" do
+    render
+    expect(rendered).to have_css("h1", text: /Move form/)
```

Comment:
This should fix the other test failures:

```suggestion
    expect(rendered).to have_css("h1", text: /Move this form/)
```